helpers/z_schema.js

```diff
@@ -36,6 +36,10 @@ var liskFormats = {
 		return /^[0-9]+$/g.test(str);
 	}, 
 
+	additionalData: function (str) {
+		return Buffer.from(str).length <= 64;
```

Comment:
would be nice if we use schema max length constant defined in the transfer file.